dicom-archive/DICOM/DICOM.pm

```diff
@@ -247,14 +404,32 @@ sub setElementValue {
   $elem->setValue($newvalue);
 }
 
-#  ------------------------------------------------------------
-#  Utility Functions (non-public)
-#  ------------------------------------------------------------
+
+=pod
+
+=head3 hexadecimally()
+
+------------------------------------------------------------------
+Utility Functions (non-public)
+------------------------------------------------------------------
```

Comment:
why did u move the non-public comment to after the `hexadecimally` function? isn;t it meant to apply to all those functions below?

Reply:
also in the .md file, this looks weird:

![weird](https://user-images.githubusercontent.com/15198379/38519889-2999fc42-3c0f-11e8-853a-c688e0829566.png)

Reply:
duh... Don't know why I did this. Brain fart ;).